tests/Jellyfin.Dlna.Tests/ProfileTester.cs

```diff
@@ -0,0 +1,107 @@
+using Emby.Dlna;
+using Emby.Dlna.PlayTo;
+using MediaBrowser.Common.Configuration;
+using MediaBrowser.Controller;
+using MediaBrowser.Model.Dlna;
+using MediaBrowser.Model.IO;
+using MediaBrowser.Model.Serialization;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Jellyfin.Dlna.Tests
+{
+    public class ProfileTester
+    {
+        private DlnaManager GetManager()
+        {
+            var xmlSerializer = new Mock<IXmlSerializer>();
+            var fileSystem = new Mock<IFileSystem>();
+            var appPaths = new Mock<IApplicationPaths>();
+            var loggerFactory = new Mock<ILoggerFactory>();
+            var appHost = new Mock<IServerApplicationHost>();
+
+            return new DlnaManager(xmlSerializer.Object, fileSystem.Object, appPaths.Object, loggerFactory.Object, appHost.Object);
+        }
+
+        [Fact]
+        public void Test_Profile_Matches()
+        {
+            var device = new DeviceInfo()
+            {
+                Name = "My Device",
+                Manufacturer = "LG Electronics",
+                ManufacturerUrl = "http://www.lge.com",
+                ModelDescription = "LG WebOSTV DMRplus",
+                ModelName = "LG TV",
+                ModelNumber = "1.0",
+            };
+
+            var profile = new DeviceProfile()
+            {
+                Name = "Test Profile",
+                FriendlyName = "My Device",
+                Manufacturer = "LG Electronics",
+                ManufacturerUrl = "http://www.lge.com",
+                ModelDescription = "LG WebOSTV DMRplus",
+                ModelName = "LG TV",
+                ModelNumber = "1.0",
+                Identification = new DeviceIdentification()
+                {
+                    FriendlyName = "My Device",
+                    Manufacturer = "LG Electronics",
+                    ManufacturerUrl = "http://www.lge.com",
+                    ModelDescription = "LG WebOSTV DMRplus",
+                    ModelName = "LG TV",
+                    ModelNumber = "1.0",
+                }
+            };
+
+            Assert.True(GetManager().IsMatch(device.ToDeviceIdentification(), profile.Identification));
```

Comment:
Typically we use the three A's:

1. Arrange
2. Act
3. Assert

Can you move the assertions down to the bottom?